backend/EkoFunkcje/EkoFunkcje/Features/Attachments/DeleteAttachmentFunction.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.Extensions.Logging;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace EkoFunkcje.Features.Attachments
+{
+    public static class DeleteAttachmentFunction
+    {
+        [FunctionName("DeleteAttachment")]
+        public static async Task<HttpResponseMessage> Run(
+            [HttpTrigger(AuthorizationLevel.Anonymous, "post", Route = "DeleteAttachment/{interventionId}/{fileName}")] HttpRequestMessage req,
+            [Blob("attachments/{interventionId}/{fileName}", FileAccess.Write, Connection = Config.StorageConnectionName)] CloudBlockBlob blobToDelete,
+            ILogger log)
+        {
+            try
+            {
+                await blobToDelete.DeleteIfExistsAsync();
+                return req.CreateResponse(HttpStatusCode.OK, "Attachment successfully deleted");
+            }
+            catch (Exception e)
+            {
+                return req.CreateErrorResponse(HttpStatusCode.BadRequest, "Could not find the blob");
```

Comment:
Message lepszy Could not find the blob. W sumie to zastanawiam sie czy to nie mogloby byc asynchroniczne jednak zamiast takiego czekania na odpowiedz, Zostawmy juz tak jak jest